menu.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Menu of TH Resturant</title>
+        <link rel="stylesheet" type="text/css" href="restyle.css"/>
+        <meta charset="UTF-8">
+    </head>
+    <body>
+        <h1 >Menu</h1>
+        <div class="row">
+            <div class="col-6 col-l-1">
+                <a href="index.html">Home</a>
+            </div>
+            <div class="col"> </div>
+            <div class="col-6 col-l-1">
```

Comment:
you are using responsive columns but they are not defined in the style sheet you use. For example col-6. You should have expanded the style sheet to contain different values col-s-1 ... col-s-12 and col-l-1 ... col-l-12 and use them for repsonsiveness.